hack/release.sh

```diff
@@ -114,6 +114,92 @@ s3_url() {
 	esac
 }
 
+map_goos_to_uname() {
+	# we need to map our GOOS to uname values
+	# see https://en.wikipedia.org/wiki/Uname
+	# ie, GOOS=linux -> "uname -s"=Linux
+	GOOS=$1
+	case "$GOOS" in
+		darwin)
+			echo "Darwin"
+			;;
+		freebsd)
+			echo "FreeBSD"
+			;;
+		linux)
+			echo "Linux"
+			;;
+		windows)
+			# this is windows use the .zip and .exe extensions for the files.
+			echo "Windows"
+			;;
+		*)
+			echo >&2 "error: can't convert $GOOS to an appropriate value for 'uname -s'"
+			exit 1
+			;;
+	esac
+}
+
+map_goarch_to_uname() {
+	# we need to map our GOARCH to uname values
+	# see https://en.wikipedia.org/wiki/Uname
+	# ie, GOARCH=amd64 -> "uname -m"=x86_64
+	GOARCH=$1
+	case "$GOARCH" in
```

Comment:
Same thought here -- these might as well be one line and not create a new variable (which might modify a variable unintentionally too, since it's not `local`).